### ISSUE TITLE
Licensing Data out of date

### ISSUE THREAD

**Issue by robertbatchelor** [SCRT-HQ/PSGSuite#374]
After a coupe of hours of screwing around I believe I have hit a wall, and I assumption is due to this module not recognising the newer license types now provided through Google Workspace.

The CMDlet Get-GSUserLicense no longer returns any licenses information against users with a Google Workspace Enterprise Standard license, or Google Workspace Enterprise Standard - Archived User license. It simply returns that no license was found. Even with the -CheckAll parameter added from this issue https://github.com/SCRT-HQ/PSGSuite/issues/270

This is further confirmed, if you try to run something like the following; `Get-GSUserLicense -License "Google-Workspace-Enterprise-Standard"` you'll receive and error with all of the expected license names, which most do not exist and do not reflect what is detailed in Googles documentation https://developers.google.com/admin-sdk/licensing/v1/how-tos/products

**Comment by sccannon** [comment]
Second. This is needed.

**Comment by mattseiter-nbs** [comment]
Yes, this is greatly needed. As it is now the license commands are not useful.

**Comment by sccannon** [comment]
Bump. This is needed.

**Comment by Buenno** [comment]
I've made some changes in the rework_licensing branch. Check it out and let me know if it helps, I'll submit a PR if it does.
